src/Tribe/Tickets.php

```diff
@@ -999,14 +1005,14 @@ protected function global_stock_mode_selector( $current_option = '' ) {
 				$identifier = esc_html( $identifier );
 				$name = esc_html( $name );
 				$selected = selected( $identifier === $current_option, true, false );
-				$output .= "\t<option value='$identifier' $selected> $name </option>\n";
+				$output .= "<label for='$identifier' class='ticket_field'><input type='radio' id='$identifier' class='' name='ticket_global_stock' value='$identifier' $selected> $name </label>\n";
```

Comment:
I know this was also true previously, but we're using `$identifier` in attributes whereas it's been escaped for wider use.